SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -81,6 +105,11 @@ struct ConfigureConnection
 		Params.network.kcp.security_type = WORKER_NETWORK_SECURITY_TYPE_INSECURE;
 		Params.network.tcp.security_type = WORKER_NETWORK_SECURITY_TYPE_INSECURE;
 
+		// Unreal GDK version
+		UnrealGDKVersionPair.name = "gdk_version";
+		UnrealGDKVersionPair.version = TCHAR_TO_UTF8(*GDKVersionLoader::GetGDKVersion());
```

Comment:
Using a TCHAR_TO_UTF8 like this isn't safe as the memory can be reused - you should use a string builder instead

Reply:
This got fixed here: https://github.com/spatialos/UnrealGDK/pull/2898/files